cmd/krel/cmd/BUILD.bazel

```diff
@@ -45,9 +45,15 @@ filegroup(
 go_test(
     name = "go_default_test",
     srcs = [
+        "changelog_data_test.go",
         "changelog_test.go",
         "root_test.go",
     ],
     embed = [":go_default_library"],
-    deps = ["@com_github_stretchr_testify//require:go_default_library"],
+    deps = [
+        "//pkg/command:go_default_library",
+        "//pkg/git:go_default_library",
+        "@com_github_stretchr_testify//require:go_default_library",
+    ],
+    data = glob(["test/**"]),
```

Comment:
@justaugustus @hoegaarden do you know how I can add my test data without breaking things up with `./hack/verify-bazel.sh` (see latest test result in prow)?

Reply:
What's the issue/the error? I think, this should be correct.
I had some [issues][i] with something like that in the past, where certain paths were slightly different in the bazel sandbox then outside of one. Is there maybe a similar issue? 

[i]: https://github.com/kubernetes/kubernetes/blob/1bb68a2cde08e39875f8e4f53bf429e14438835e/staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/fixtures/fixtures.go#L53-L59

Reply:
Hm no my main issue is that the line will get removed when running:
```
> ./hack/update-all.sh
```
Do you know any way how to avoid that?

Reply:
Can you check if something like this works:
```diff
diff --git a/cmd/krel/cmd/BUILD.bazel b/cmd/krel/cmd/BUILD.bazel
index 94f3e913d..6bb2f0770 100644
--- a/cmd/krel/cmd/BUILD.bazel
+++ b/cmd/krel/cmd/BUILD.bazel
@@ -36,6 +36,7 @@ go_test(
         "changelog_test.go",
         "root_test.go",
     ],
+    data = [":testdata"],
     embed = [":go_default_library"],
     deps = ["@com_github_stretchr_testify//require:go_default_library"],
 )
@@ -53,3 +54,9 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+filegroup(
+    name = "testdata",
+    srcs = glob(["test/**"]),
+    visibility = ["//visibility:private"],
+)
```


This at least does not get removed on a `./hack/update-all.sh` -- "works on my machine"™️

Alternatively, we could just get rid of bazel. Yes, I mean it.

Reply:
> Alternatively, we could just get rid of bazel. Yes, I mean it.

:see_no_evil:

Thanks, I'll check if that works.